Bootstrap.php

```diff
@@ -968,8 +966,7 @@ protected function validateEvent($controller, $module, $ctrl = null, $action = n
      */
     public function onPostDispatchFrontend(Enlight_Controller_ActionEventArgs $args)
     {
-        if (!$this->shopHasConnectedAccount()
```

Comment:
I think we can leave this guard.

Reply:
It breaks the backend, due to the `shop` service not being initialised in the container. That is why the whole backend becomes inaccessible. The only reason it did not happen on older version of the module is because it was a second OR argument, so it was never reached.

Reply:
good one!